assets/js/components/Dashboards/dashboards.gql.mock.js

```diff
@@ -77,6 +78,31 @@ export const mockGetBatchesResults = [
   },
 ];
 
+export const mockPreservationChecks = [
+  {
+    id: "7c171c70-1f1a-4db8-8dcb-07c2c0003e10",
```

Comment:
Same here, just to make code reading fast, if we can alphabetize that'd be great.